templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete car</h1>
+  <p>Are you sure, you want to delete {{ car }}?</p>
+  <form method="post">

```

Comment:
Consider adding an `action` attribute to the form tag to explicitly specify the URL for the delete action. This can help prevent issues if the template is reused or modified.